app/validators/representation_order_validator.rb

```diff
@@ -14,8 +14,8 @@ def self.fields
   # must not be earlier than the first rep order date
   # must not be earlier than the earliest permitted date
   def validate_representation_order_date
-    validate_presence(:representation_order_date, 'blank')
-    validate_on_or_before(Date.today, :representation_order_date, 'in_future')
+    validate_presence(:representation_order_date, :blank)
```

Comment:
This modified file relates to the `defendants` form_step, is this required for `supporting_evidence`?